shape_completion_training/src/shape_completion_training/model/filepath_tools.py

```diff
@@ -1,12 +1,21 @@
-
 from __future__ import print_function
 import os
+import time
 from os.path import join
 import shutil
 import datetime
 import subprocess
 import json
 
+import git
```

Comment:
I am willing to switch, but I can't test this out yet.
In general, I'd prefer this change to be a separate commit from formatting

Reply:
gitpython works fine.
The tradeoff is adding another dependency instead of a `subprocess` call. Not really sure it is worth it, but no big deal.

Reply:
well this is actually doing something different, I didn't replace your subprocess calls (yet). I just use this to get the git hash.